server/utils/urlAccessByStatus.ts

```diff
@@ -28,6 +29,7 @@ const allowedPaths = [
     allowed: [
       '/licence/create/id/(\\d)*/check-your-answers.*',
       '/licence/create/id/(\\d)*/edit.*',
+      '/licence/approve/id/(\\d)*/confirm-rejected.*',
```

Comment:
Integration tests will not have caught this if wiremock has not been told to return REJECTED status on the confirmation page ... sometimes I wonder if we need regression tests which run on the full stack ... an overnight selenium job or something ?